src/utils.ts

```diff
@@ -454,6 +454,7 @@ export function verifyLang(lang?: string): boolean {
   case "zh-TW":
   case "pt-BR":
   case "ko":
+  case "ja":
     return true;
   default:
```

Comment:
```suggestion
    return true;
  case "ja":
  default:
```

`verifylang()` needs `public/images/types_ja.png` and `types_ja.json`.
Setting return value to `false` is now better, until type images are added.

Reply:
+ `battle-stat.test.ts` is for `npm test`, checking `locales/ja/pokemon-info.ts` and `locale/ja/battle.ts`